### DISCUSSION TITLE
AWS Backup feature updates

### DISCUSSION THREAD

**Discussion by aaroncarlucci** [superwerker/superwerker#514] [Ideas]
After a recent review of the AWS Backup documentation in #284, we've generated a brainstorm of potential improvements to the AWS Backup implementation. Here is a list for discussion:

1. Add support for new AWS services
    - [Aurora](https://aws.amazon.com/blogs/storage/using-aws-backup-to-protect-amazon-aurora-databases/)
    - [DocumentDB](https://aws.amazon.com/blogs/storage/manage-backup-and-restore-of-amazon-documentdb-with-aws-backup/)
    - Support for [S3](https://aws.amazon.com/about-aws/whats-new/2022/02/general-availability-aws-backup-amazon-s3/) is available, but initial recommendation is to exclude it for cost reasons
1. EFS should now be supported by AWS Config, so add it to the conformance pack
1. Enable [continuous backup](https://docs.aws.amazon.com/aws-backup/latest/devguide/point-in-time-recovery.html) for supported services (RDS, Aurora and investigate DynamoDB PITR support)
1. Enable [Advanced DynamoDB Backup](https://docs.aws.amazon.com/aws-backup/latest/devguide/advanced-ddb-backup.html)
1. Delegate AWS Backup administration to the Audit account, allowing it to modify backup policies and observe backup activities

Putting these features out for discussion amongst the community and stakeholders. Any comments, questions or concerns?

**Comment by peermuellerxw** [comment]
EFS is not supported by AWS config [required-tags ](https://docs.aws.amazon.com/config/latest/developerguide/required-tags.html) rule, unfortunately. Same goes for Aurora and DocumentDB.

We could research if there is a another way to trigger the remediation and thus setting the tags automatically.
